index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/1935d064dd.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;200;300;400;500;600;700;800;900&family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portifolio</title>
+  </head>
+  <body>
+    <header class="header-class">
+      <a href="index.html" id="logo">Filbert</a>
+      <nav id="nav-bar">
+        <i class="fa fa-bars" id="bars"></i>
+      </nav>
+    </header>

```

Comment:
Nice job so far making your `header` section semantic😉. Let's try to take away the white line below the toolbar so it matches the design😉.

yours
<img width="344" alt="image" src="https://github.com/FilbertKimasa/Portfolio/assets/64643995/616494b4-b946-41f4-adbc-3e3423d49171">

I probably think you are mistaking the sketch lines to be part of the design😉. Kindly take it away as that is not part of the deisgn👍.